jina/orchestrate/flow/base.py

```diff
@@ -1181,6 +1195,13 @@ def add(
                 'External Executors with multiple needs have to do auto reduce.'
             )
 
+        args.external_replica_ports = make_iterable(_parse_ports(str(args.port)))
```

Comment:
do not pass this as `args` to `Deployment`. `Deployment` itself should do this parsing

Reply:
Other args like workspace and random port assignment get also handled and changed here, then passed to the deployment as args, so I wanted to follow that pattern.
But I can of course change it if you want.

Reply:
but this seems like `new` arguments, that do not match `CLI` params

Reply:
fair enough, will change it